release-notes/opensearch-release-notes-2.7.0.md

```diff
@@ -0,0 +1,641 @@
+
+### New Features
+* You can now choose segment replication as a data replication strategy for production workloads. Segment replication can yield improved indexing throughput and lower resource utilization at the expense of increased network utilization and refresh times.
+* Searchable snapshots also move from experimental to production-ready with this release, allowing users to search indexes that are stored as snapshots within remote repositories in real time.
+* The third feature moving out of experimental status is support for multiple data sources in OpenSearch Dashboards. Dynamically manage data sources across multiple OpenSearch clusters, combine visualizations into a single dashboard, and more.
+* New flat object fields let you store complex JSON objects in an index without indexing all subfields separately, offering improved resource utilization.
+* You can now access observability features from OpenSearch Dashboards, create your own observability dashboards, add event analytics visualizations, and more without leaving the Dashboards environment.
+* Geospatial functionality receives upgrades as new shape-based filters let you filter your geospatial data by drawing a rectangle or polygon over a selected area of the map. This release also enables OpenSearch to display maps in local languages.
+* OpenSearch makes managing multiple indexes simpler with the addition of component templates. These templates can help you abstract common index settings, mappings, and aliases into reusable building blocks.
+* Another time-saving upgrade for OpenSearch administrators comes with the availability of dynamic tenant management in OpenSearch Dashboards. Admins can view, configure, and enable or disable tenancy within Dashboards and effect those changes without a restart.
+* You can now use the Performance Analyzer plugin to identify hot shards within an index, so you can take action and mitigate potential impacts on performance and availability.
+
+### Experimental Features
+OpenSearch 2.7.0 includes the following experimental features. Experimental features are disabled by default. For instructions on how to enable them, refer to the documentation for the feature.
+
+* A new correlation engine is added to the Security Analytics toolkit, enabling a knowledge graph that can be used to identify, store, and recall connected events data to help you identify patterns and investigate relationships.
+* The experimental ML Framework gains a new automatic reloading mechanism for ML models, allowing you to auto-reload deployed models when a cluster restarts or when a node rejoins the cluster.
+
+### OpenSearch Ml Commons Dashboards
```

Comment:
should be "ML Commons" (with a capital L). And I think the section header can say "Change to ML Commons Dashboards" or "ML Commons Dashboards Update"